lib/pbench/server/api/resources/endpoint_configure.py

```diff
@@ -0,0 +1,99 @@
+from flask_restful import Resource, abort
+from flask import jsonify
+
+from pbench.server.api.resources.query_apis import get_index_prefix
+
+
+class EndpointConfig(Resource):
+    """
+    EndpointConfig API resource: this supports dynamic dashboard configuration
+    from the Pbench server rather than constructing a disconnected dashboard
+    config file.
+    """
+
+    def __init__(self, config, logger):
+        """
+        __init__ Construct the API resource
+
+        Args:
+            config (PbenchServerConfig): server config values
+            logger (Logger): message logging
+        """
+        self.logger = logger
+        port = config.get("pbench-server", "rest_port")
+        host = config.get("pbench-server", "host")
+        self.host = f"{str(host)}"
```

Comment:
Wait, why not just, `self.host = str(host)`?

Reply:
Because ... I just kinda made the change on autopilot without really thinking? It does look just a teeny bit... weird, doesn't it.